src/java/org/apache/cassandra/db/IMutation.java

```diff
@@ -24,24 +24,25 @@
 import org.apache.cassandra.config.DatabaseDescriptor;
 import org.apache.cassandra.db.partitions.PartitionUpdate;
 import org.apache.cassandra.schema.TableId;
+import org.apache.cassandra.service.ClientState;
 
 public interface IMutation
 {
-    public long MAX_MUTATION_SIZE = DatabaseDescriptor.getMaxMutationSize();
+    long MAX_MUTATION_SIZE = DatabaseDescriptor.getMaxMutationSize();
 
-    public void apply();
```

Comment:
if you are removing this, we should be consistent and remove it for `void validateSize` and `static long dataSize` methods at the end of this file as well

Reply:
Done